### PR TITLE
[Editor] Fix the CSS properties of the canvas when it's used in a stampEditor (bug 1895909)

### PR DESCRIPTION
**Pull request by calixteman**
And move the page canvas properties under canvasWrapper in order to avoid future regressions.

### CODE REVIEW

**Comment by Snuffleupagus** [comment]
Can we please just revert the regression patch *first*, see PR #18079, so that we get back to a working state before making any further changes to this code?

File: test/integration/stamp_editor_spec.mjs [hunk 1]
```diff
@@ -698,4 +699,67 @@ describe("Stamp Editor", () => {
...
+
+    beforeAll(async () => {
+      pages = await loadAndWait("tracemonkey.pdf", ".annotationEditorLayer");
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that a stamp can be undone", async () => {
```
Comment (timvandermeij) [inline test/integration/stamp_editor_spec.mjs:714]: Shouldn't this be "must check if a stamp can be resized", or something similar (I presume this was copy/pasted)?

**Comment by calixteman** [comment]
I added back the `display: block` for the page canvas in order to avoid any confusion and @nicolo-ribaudo can make a follow-up patch to fix the zoom stuff.

**Review by Snuffleupagus — APPROVED** [review]
r=me, with passing Windows tests; thank you!

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Thank you for improving the CSS here, and adding more tests to avoid future regressions!

**Pull request merged**